src/cares_wrap.cc

```diff
@@ -1935,6 +1935,30 @@ void IsIPv6(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+void canonicalIP(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Style: `CanonicalIP`

As well, I'd call it `CanonicalizeIP` for two reasons:

1. It makes it clear it's an imperative, not a predicate (i.e., it transforms, it doesn't just check.)
2. `lI` is almost indistinguishable from `ll`, `II`, etc. with some fonts.  (Minor thing.  Still.)